pipeline-modules/app-service/gcp/custom-step.yaml

```diff
@@ -0,0 +1,62 @@
+provisioner: gcp
+name: custom-step
```

Comment:
custom-step sounds too generic. It is not clear what this is going to do. I would then just call it docker-exec.

Reply:
I don't want to belabour this point but I tend to agree with @petmal